src/App.php

```diff
@@ -198,6 +199,16 @@ function () use ( $slim ) {
 					$page( 'callback' );
 				} )->name( 'oauth_callback' );
 			} );
+		$slim->get( '/index.css',
+			function () use ( $slim ) {
+				// Compile LESS if need be, otherwise serve cached asset
+				// Cached files get automatically deleted if they are over a week old
+				$lessFiles = array( APP_ROOT . '/src/Less/index.less' => '/copypatrol/' );
```

Comment:
What does this line do? I'm guessing the first part is the source path of the file but the second? And hardcoding it here is not a good idea. It didn't work for me, routing issues. And it'll break our test instance /plagiabot. We can go with a new env variable to handle this.